src/lib/metaphysics.ts

```diff
@@ -1,9 +1,12 @@
+import { uncrunch } from "graphql-crunch"
 import { NativeModules } from "react-native"
 const Emission = NativeModules.Emission || {}
 
 import { metaphysicsURL } from "./relay/config"
 import { NetworkError } from "./utils/errors"
 
+const enableCrunch = Emission.crunch != null ? Emission.crunch : "true"
```

Comment:
Is there a specific reason we need this to be an option at all?

Reply:
Likely not. I noticed that some values below were being pulled from NativeModules so I added it as a potential configuration point. In hindsight that might be better served as an env variable. 

Would you like me to remove it?